modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/LayoutExportController.java

```diff
@@ -214,6 +214,12 @@ protected File doExport(PortletDataContext portletDataContext)
 		headerElement.addAttribute(
 			"build-number", String.valueOf(ReleaseInfo.getBuildNumber()));
 
+		Group group = _groupLocalService.fetchGroup(
+			portletDataContext.getGroupId());
+
+		headerElement.addAttribute(
+			"group-friendly-url", group.getFriendlyURL());
```

Comment:
I would move this closer to other group related attributes like group-id down below.